src/components/PersonCardList/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const PersonCardList = ({ data }) => {
+  console.log("data PersonCardList", data);
+
+  return data.map((item, index) => (
```

Comment:
you can destructuring `item` like this `data.map(({img, name, role, link}, index) => (`